internal-tooling/performanceUtils.ts

```diff
@@ -0,0 +1,108 @@
+/*!
+ * Copyright 2022 Google LLC. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {execSync} from 'child_process';
+import {mkdirSync} from 'fs';
+import path = require('path');
+import * as uuid from 'uuid';
+
+export const BLOCK_SIZE_IN_BYTES = 1024;
+export const DEFAULT_SMALL_FILE_SIZE_BYTES = 5120;
+export const DEFAULT_LARGE_FILE_SIZE_BYTES = 2.147e9;
+
+const CREATE_DIRECTORY = 1;
+
+/**
+ * Create a uniformly distributed random integer beween the inclusive min and max provided.
+ *
+ * @param {number} minInclusive lower bound (inclusive) of the range of random integer to return.
+ * @param {number} maxInclusive upper bound (inclusive) of the range of random integer to return.
+ * @returns {number} returns a random integer between minInclusive and maxInclusive
+ */
+export function randomInteger(minInclusive: number, maxInclusive: number) {
+  // Utilizing Math.random will generate uniformly distributed random numbers.
+  return (
+    Math.floor(Math.random() * (maxInclusive - minInclusive + 1)) + minInclusive
```

Comment:
this will go over maxInclusive if Math.random() returns 1, right?
say max is 10 and min is 1.
This becomes (1* (10-1+1))+1 = 11

Reply:
Math.random is [exclusive](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/random) of 1.